help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -158,39 +189,117 @@ To configure details for the destination, fill in the fields below.
 
 >[!CONTEXTUALHELP]
 >id="platform_destinations_liveramp_distribution_iheartmedia_client"
->title="Client name"
+>title="Client Name"
 >abstract="Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
 
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the iHeartMedia destination.](../../assets/catalog/advertising/liveramp-distribution/LR_iHeart_DestSpecific.png)
+
+* **[!UICONTROL Client Name]**: Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
+
 ### [!DNL Index Exchange] {#index-exchange}
 
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_index_advertiseraccountname"
+>title="Account Name"
+>abstract="Your Index Exchange client account name. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Index Exchange destination.](../../assets/catalog/advertising/liveramp-distribution/LR_IndexExchange_DestSpecific.png)
+
+* **[!UICONTROL Account Name]**: Your Index Exchange client account name. Do not use spaces or special characters.
+
 ### [!DNL Magnite CTV Platform] {#magnite}
 
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitectv_client"
+>title="Client"
+>abstract="Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
+
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Magnite CTV destination.](../../assets/catalog/advertising/liveramp-distribution/LR_MagniteCTV_DestSpecific.png)
+
+* **[!UICONTROL Client]**: Your Client name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
+
 ### [!DNL Magnite DV+ (Rubicon Project)] {#magnite-dv}
 
-### [!DNL One Fox] {#fox}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_partnerid"
+>title="PartnerId"
+>abstract="The Rubicon Project Partner ID associated with the publisher who owns the segment/data. Please contact your Rubicon Project account representative if unsure of the value you should use."
 
-### [!DNL Pandora] {#pandora}
+>[!CONTEXTUALHELP]
+>id="platform_destinations_liveramp_distribution_magnitedv+_seatid"
+>title="SeatId"
```

Comment:
>title="SeatId"

The UI field labels should not read like code variables. This should be called "Seat ID" in the customer data fields, labels, and the popover. Please update this to the correct formatting.